src/test/java/org/jenkinsci/plugins/shortwspath/LocatorTest.java

```diff
@@ -54,8 +63,11 @@ public void doNothingIfThereIsEnoughRoom() throws Exception {
         FreeStyleProject p = f.createProject(FreeStyleProject.class, "and_a_project_in_it");
         p.setAssignedNode(s);
 
+        // Not enough for anything.
+        setMaxPathLength(s, 4096);
+
         FreeStyleBuild b = p.scheduleBuild2(0).get();
-        assertThat(b.getWorkspace().getRemote(), equalTo(s.getRootPath() + "/workspace/" + p.getFullName()));
+        assertThat(b.getWorkspace().getRemote(), equalTo(formatPath(s.getRootPath() + "/workspace/" + p.getFullName())));
```

Comment:
Why is this needed? If I am not mistaken, this is implemented as `File.createTempFile("hudson", "test", base);` that should be Windows friendly.